.coafile

```diff
@@ -0,0 +1,41 @@
+[Default]
+ignore = .tox/**, .gitignore, .gitreview, .gitmodules, **/provisioning/roles/**
```

Comment:
Ah, missed tox question. Yes, I guess we can remove that too, we use Rakefile for that job in puppet